### DISCUSSION TITLE
Hi! Today is my first day learning about Ethereum

### DISCUSSION THREAD

**Discussion by NazrinKhan** [Cyfrin/foundry-full-course-cu#1621] [General]
Just finished lesson 1 of Solidity

**Comment by EngrPips** [comment]
Hello @NazrinKhan, Welcome aboard, and I wish you the best on your journey.

**Reply by NazrinKhan** [reply]
:))

**Comment by NitishRawat123** [comment]
Congratulations!!! btw are you also sharing and writing somewhere along?

**Reply by NazrinKhan** [reply]
currently I'm saving my codes in my desktop but eventually I upload my projects in github

**Comment by TheDuyIT** [comment]
I wish all the best to you

**Reply by NazrinKhan** [reply]
Thank you so much
